src/main/java/com/sap/sgs/phosphor/fosstars/model/rating/NotApplicableLabel.java

```diff
@@ -0,0 +1,38 @@
+package com.sap.sgs.phosphor.fosstars.model.rating;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.sap.sgs.phosphor.fosstars.model.Label;
+
+/**
+ * This is a label for a score value that is marked as not-applicable.
+ */
+public class NotApplicableLabel implements Label {
+
+  @Override
+  @JsonIgnore
+  public final String name() {
+    return "N/A";
+  }
+
+  @Override
+  @JsonIgnore
+  public final boolean isNotApplicable() {
+    return true;
+  }
+
+  @Override
+  public boolean equals(Object obj) {
```

Comment:
Do we really need equals and hashcode for this class?

Reply:
I think so. With the current implementation of this class, all instances should be equal. `Object.equals()` doesn't guarantees it.